CMSgov/cmcs-eregulations
# EREGCSC-1577 BASE_URL in app; removing hard-coded prod domain name

Resolves [EREGCSC-1577](https://jiraent.cms.gov/browse/EREGCSC-1577)

**Description**
The Resources test suite for Cypress was failing only when being run against production due to the AWS URL having a subdirectory (`random-guid.amazonaws.com/prod`) and the custom domain name not having one (`regulations-pilot.cms.gov`).  This was due to how we were setting the `base` property of the Vue router.  This has been fixed.

**This pull request changes**

- Pulls `CUSTOM_URL` from AWS Parameter Store to be used as canonical custom domain name
- Compares `CUSTOM_URL` with `host` (provided by Django) to properly set Vue Router logic on prod

**Steps to manually verify this change**

2. Deploy to prod and check that Resources test suite passes and the site works as expected
